### DISCUSSION TITLE
[Request] Hide incomplete shows in Progress screen

### DISCUSSION THREAD

**Discussion by rkl92** [trakt/showly#848] [Q&A] [answered]
Hello. I'm running into an annoying issue (for me at least it's an issue) where I do not watch all the seasons of a show, for example, I skipped season 2 of True Detective and I do not want it to show on my Shows Progress because I did not check season 2 as seen. This also goes for shows where I've watched season 1 and then didn't follow up on it, these still show on my Progress page. I know I can remove these shows from "My shows" and leave only whatever I want checked as seen, but then there's no screen where these are tracked as Incomplete or something along this line, no icon in the top-right corner etc, it's like I never watched any of that show unless I click on it to get to the show page and see there that I've checked season 1 in the past. Or an option to show in Progress only the future episodes. I dunno, to me, it would be a great addition to an already awesome app. I just want to track incomplete shows that I've seen and also keep my Progress page clean.

Thanks!

**Comment by blackholeearth** [comment, marked as answer]
**ONHOLD**:
Shows i don't want to see in my progress  . (İ will deal with them later)

1) Long tap on show.    (You will see menu.)
2) Press onhold.   ( İt will move down.)

(note: if you delete and reinstall app, shows that are onhold will goback to progress )

![Screenshot_20240423-003003_Showly](https://github.com/michaldrabik/showly-2.0/assets/7512105/b81e3518-9152-436b-8400-84b8cc045197)


**DROPPED**:
İ have watched s1 , i will not watch s2, s3 or any future season of this show.

1) At the same menu,  
2) You need to press HİDDEN

**Comment by jpsnyder** [comment]
In the settings you can set "Next Episode" to "Last watched episode" instead of "Oldest unwatched episode".
If you do this, when you track you have watched the most recent episode of a show, the show won't appear in your show list again until a new episode airs. The progress bar still shows you haven't watched that missing season, but I think I prefer that. This is very helpful for tracking shows with a huge back log of seasons that you don't care to watch.
